package.json

```diff
@@ -81,7 +81,7 @@
     }
   },
   "engines": {
-    "node": "12.16.3"
+    "node": "12.16.1"
```

Comment:
why change this?

Reply:
I have Node 12.6.11 and it wouldn't work with this project. Should I change it to just 12.6?

Reply:
no you use nvm and install 12.16.3